styles.css

```diff
@@ -78,3 +87,199 @@ div a img {
   height: 3px;
   background-color: #6070ff;
 }
+
+.project-section {
+  padding: 5% 5%;
+  gap: 50px;
+  background-color: #e5e5e5;
+}
+
+.project-items {
+  background-color: white;
+  border-radius: 24px;
+  border: 1px solid #dfe1e6;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-top: 100px;
+}
+
+.project-info {
+  padding: 20px;
+}
+
+.project-info h2 {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-top: 12px;
+  color: #344563;
+}
+
+.project-details {
+  display: flex;
+  padding-top: 15px;
+  justify-content: space-between;
+  color: #344563;
+}
+
+.project-details .project-detail {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.project-details .project-stack {
+  display: flex;
+  width: 100%;
+  justify-content: space-evenly;
+}
+
+.project-stack li {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0;
+  color: #afaaaa;
+}
+
+.project-info p {
+  text-align: left;
+}
+
+.project-info .project-tech {
+  display: flex;
+  padding-top: 15px;
+}
+
+.project-tech span {
+  padding: 6px 12px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  border-radius: 8px;
+  margin-right: 10px;
+  color: #6070ff;
+  border: none;
+  background: #ebebff;
+}
+
+.project-info .project-button {
+  padding-top: 40px;
+}
+
+.project-info a {
+  border: 1.5px solid #6070ff;
+  border-radius: 16px;
+  color: #6070ff;
+  padding: 16px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
+
+.about-section {
```

Comment:
- [ ] as mentioned in the requirements, this section should be aligned with flexbox , kindly add that